docs/how-to/mining.md

```diff
@@ -0,0 +1,78 @@
+
+**Retrieval miners** are the veins of the network. They earn Filecoin by winning bids and mining fees for a particular file, which are determined by the market value of said file's size. A retrieval miner’s bandwidth and bid/initial response time for deals (i.e., latency and proximity to clients) will determine its ability to close retrieval deals on the network. The maximum bandwidth of a retrieval miner will set the total quantity of deals it can make.
+
+In the current implementation, the focus is mostly on storage miners, which sell storage capacity in exchange for FIL. The below sections discuss how to initiate, maintain and receive rewards for a storage miner.
+
+### Coming from another ecosystem?
+<-- TBD -->
+
+## Hardware recommendations
+
+The current recommended system specifications for operating a storage miner are:
+
+* An 8+ core CPU
+* A NVIDIA-manufactured GPU chip (to be expanded on)
+* A dedicated SSD to act as a large cache store (512GB+)
+* A large amount of RAM for computing data replication (128GB+)
+
+### Aren't these requirements comparatively high?
+
+<-- still need to think about how to phrase this -->
+
+## Getting started as a miner
+
+For instructions on getting the Filecoin network running and a miner initialized, see the [Lotus client manual](https://lotu.sh/).
+
+## All about deals
+
+<-- bckground -->
+
+<-- instructions, setting up ask - need advising here -->
+
+By default, all Filecoin miners are set to auto-accept any deal that meets their criteria. For those that wish to curate the deals being stored on their system, it is possible to disable the auto-accepting feature with:
+<-- need lotus instructions-->
+
+For comprehensive information on how deals are performed on the Filecoin network, see [this section of the documentation](https://docs.filecoin.io/how-to/store-making-storage-deals/).
+
+## Getting rewards
+
+In Filecoin, miners are able to received two different types of rewards for their efforts: storage fees, and block rewards.
+
+**Storage fees** are the fees paid regularly by clients after a deal has been reached. These fees are automatically deposited into a miner's associated withdrawl wallet as they continue to perform their duties over time, and  are briefly locked upon being received <-- until certain confirmation count or? -->
+
+**Block rewards** are large sums that are given to the miner credited for a new block. Unlike storage fees, these rewards do not come from an associated client; rather, the network "prints" new FIL as both an inflationary measure and an incentive to miners advancing the chain. All active miners on the network have a chance at recieving a block reward, their chance at such being directly proportional to the amount of storage space currently being contributed to the network.
```

Comment:
@arajasek @zx @ianjdarrow can you confirm that all miners can receive a block reward? I believe there's a minimum threshold but could be mistaken.